test/framework/src/main/java/org/elasticsearch/test/ESTestCase.java

```diff
@@ -426,6 +428,14 @@ private void ensureNoWarnings() {
         }
     }
 
+    protected void ensureNoDeprecationOnSecurityPlugin() {
```

Comment:
Why do we add this to `ESTestCase` when it's only used in `SecurityPluginNoDeprecationTests` ?

Reply:
After refactoring `getRestHandlerWrapper` to not fail when SSLService is not created was able to move it to the SecurityTests. Reverting the Integ Test changes